plugins/formal-verification/formver.plugin/src/org/jetbrains/kotlin/formver/VerifierErrorInterpreter.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2010-2023 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.formver
+
+import org.jetbrains.kotlin.diagnostics.KtDiagnosticFactory1
+import org.jetbrains.kotlin.formver.info.SourceRole
+import org.jetbrains.kotlin.formver.viper.errors.*
+import org.jetbrains.kotlin.utils.addToStdlib.ifTrue
+
+data class HumanReadableMessage(val pluginError: KtDiagnosticFactory1<String>, val extraMsg: String? = null)
+
+object VerifierErrorInterpreter {
+
+    private fun ConsistencyError.toHumanReadableMessage(debug: Boolean): HumanReadableMessage = when (debug) {
```

Comment:
I think using whatever flag system you intend to use for the gradle plugin would be a better choice here rather than a blanket `debug` flag.

Reply:
I agree! I'm working on the flag in a different branch. I will open a PR when it will be done.